symfony.lock

```diff
@@ -268,6 +268,9 @@
     "symfony/http-kernel": {
         "version": "v5.1.2"
     },
+    "symfony/lock": {
+        "version": "v5.1.8"
```

Comment:
Shouldn't it be 5.2?

Reply:
I am not sure. I did not modify this myself. 

I dont think it matters too much.